### DISCUSSION TITLE
Installing on Onyx BOOX Android 2.3.1 (Gingerbread)

### DISCUSSION THREAD

**Discussion by twisteros** [koreader/koreader#7382] [Q&A]
Any chance to install on Android 2.3.1 (Gingerbread)?

I tried to install the latest via ADB and got "Failure [INSTALL_FAILED_OLDER_SDK]"

**Comment by Frenzie** [comment]
Unfortunately not. We were forced to drop support a while back.

**Comment by twisteros** [comment]
Bugger
Do you remember roughly when was that?
I'd look for a release from that time.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Reply by Frenzie** [reply]
I think that might be 2018.12 but don't quote me on that.

https://github.com/koreader/koreader/releases/tag/v2018.12

**Comment by twisteros** [comment]
Got in installed on Gingerbread.
Didn't work though.
Thanks anyway !

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Reply by Frenzie** [reply]
Weird, the changelog says I fixed some 2.3 specific issues. Onyx isn't exactly standard though.

**Comment by czosnkowy666** [comment]
I have same error message on inkBook with android 4.2.2 and after some trys and errors last version I could install succefuly is [2023.06.1](https://github.com/koreader/koreader/releases/tag/v2023.06.1) as in next one there is NDK upgrade from 15 to 23 ( [#10610](https://github.com/koreader/koreader/pull/10610) )
